### DISCUSSION TITLE
How to enable display of zeros?

### DISCUSSION THREAD

**Discussion by ForumLiker** [dbeaver/dbeaver#11317] [Q&A] [answered]
#### System information: 
- Operating system (distribution):Arch Linux
- DBeaver version: 7.1.5

#### Connection specification:
- Database name and version: MariaDB 10.5.5

#### Describe your question:
Hi everyone. d1 - **decimal(9,4)**; d2 - **float(9,4)**; d3 - **double(9,4)**
`SELECT SUM(d1) AS a, SUM(d2) AS b, SUM(d3) AS c FROM number_test;`
![DeepinScreenshot_выберите-область_20200830225421](https://user-images.githubusercontent.com/11502792/91668219-c463c400-eb13-11ea-8b3c-5b37f876cf27.png)
![DeepinScreenshot_выберите-область_20200830225458](https://user-images.githubusercontent.com/11502792/91668233-db0a1b00-eb13-11ea-83ac-a38c21156ae0.png)
Where has zero gone?
And, in the table where **int(3) zerofill**
![DeepinScreenshot_выберите-область_20200830225701](https://user-images.githubusercontent.com/11502792/91668263-20c6e380-eb14-11ea-80a0-9a02ae349216.png)
![DeepinScreenshot_выберите-область_20200830225729](https://user-images.githubusercontent.com/11502792/91668270-30462c80-eb14-11ea-8652-81d1a50fff84.png)
Where are the zeros?

**Comment by kseniiaguzeeva** [comment]
Thank you for the report.

P.S._Table Kseniia.number_test is created to test_

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please enable Use native numeric format 
![dbeaver_vKU2pObDLF](https://user-images.githubusercontent.com/51405061/92215851-13a44e80-ee9e-11ea-8147-9f31f93e5194.png)

**Comment by ForumLiker** [comment]
@kseniiaguzeeva this way worked with only one data type
![DeepinScreenshot_выберите-область_20200905220046](https://user-images.githubusercontent.com/11502792/92311828-4300ab80-efc3-11ea-9f81-c7292903c491.png)

**Comment by kseniiaguzeeva** [comment]
Seems, it is MariaDB specific behaviour. Please see https://stackoverflow.com/questions/54283126/double-in-mariadb
